Controller/InputProcessor.cs

```diff
@@ -1,7 +1,67 @@
+using HashPhotoSlideshow.Model;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
 namespace HashPhotoSlideshow.Controller
 {
     public class InputProcessor
     {
-        //first commit
+        public List<Photo> _photo;
+
+        public InputProcessor()
+        {
+            _photo = new List<Photo>();
+        }
+
+        public void ReadInputFile()
+        {
+            //string path = "..\\..\\..\\inputFile.txt"; 
+            string path = "inputFile.txt";
```

Comment:
Your function should take in a string path parameter? Remove this hardcoded path variable.